src/flows/parser_item_html_text_streaming_flow.py

```diff
@@ -133,26 +163,79 @@ def load(key: str):
     uri = f"s3://{S3_BUCKET}/{key}"
     logger = prefect.context.get("logger")
     logger.info(f"Snowflake loading key: {uri}")
-    return SnowflakeQuery(**SNOWFLAKE_DEFAULT_DICT).run(data={'uri': uri}, query=IMPORT_SQL)
+    return SnowflakeQuery(**SNOWFLAKE_DEFAULT_DICT).run(
+        data={"uri": uri}, query=IMPORT_SQL
+    )
 
 
 @task()
 def cleanup(key: str):
     bucket = S3_BUCKET
     logger = prefect.context.get("logger")
     logger.info(f"deleting file: {str(key)}")
-    s3 = boto3.resource('s3')
+    s3 = boto3.resource("s3")
     s3.Object(bucket, key).delete()
 
 
-with Flow(FLOW_NAME, executor=LocalDaskExecutor(), schedule=get_interval_schedule(minutes=1440)) as flow:
+@task()
+def ordered_dataset_insert_transform() -> None:
+    """This task will incrementally insert new records into the article_content_ordered_live table.
+    This table was created manually from existing raw data as a starting point.
+    Query is built so that this task is itempotent.
+    """
+    logger = prefect.context.get("logger")
+    logger.info("Adding new records to article_content_ordered_live dataset...")
+    result = SnowflakeQueriesFromFile(**FLOW_SNOWFLAKE_DICT).run(
```

Comment:
Are parameters passed to the constructor sent to Prefect Cloud? If so, we should pass `FLOW_SNOWFLAKE_DICT` to the run method.

Reply:
Looking at an arbitrary task that passes arguments to the constructor, I don't see them in Prefect Cloud, so ignore the above.

Maybe back in the day when we were using S3 for Flow storage the Task constructor arguments were stored on S3? In any case, I think we're ok.